Documentation/UserGuide.md

```diff
@@ -1,27 +1,86 @@
-# User Guid for p, the genral project manager
-Many languages have an own project manager.
+# User Guide for p, the genral project manager
```

Comment:
"general"

Reply:
Fixed it with https://github.com/boxed/p/pull/6/commits/e11789eb376786aa75653a60f616747278678504
And removed Emacs save files with https://github.com/boxed/p/pull/6/commits/66c5af73ba64149fcc0181e5f3f45dc7ea38a79e